src/app/projects/page.tsx

```diff
@@ -10,27 +11,52 @@ import {
   CardTitle,
 } from "~/components/ui/card";
 
+const project_statuses = ["all-projects", "archived-projects"];
```

Comment:
Why do we distinguish between `all` and `archived` instead of `all`, `pending`, `in progress`, and `done`? It's a secondary thing, but if we are going to have different status for each project, we could also stick to the 4 status version? Correct me if I understood incorrectly.
Also, if we go for this two status, what does archived mean? Will an archived project be shown in the all project view? And should we also define a button or so to archive/unarchive a project?

Reply:
in the meeting we discussed that archived would only be meant for projects that the assessor doesn't wish to see on the main projects page. 

We also mentioned that we do not necessarily have a notion for what makes a project pending or in progress or done, etc. Because the assessment does not reflect if the project itself is done in terms of development or not. New features can change the assessment's reality. So we shouldn't be determining the project's progress from a Refa point of view.

Reply:
I see, then maybe add a functionality for assessor (or actually for all users) to archive or unarchive a project would be helpful.

Or we just display all project without distinguish them.

More thoughts for notion on project status: yes we don't have it now, but later for some progress/assessment metrics charts on the project details page we would probably have to define it anyway. Or maybe we don't have to, not sure lol. But it is not a primary thing

Reply:
Yep, we could add that functionality, although this requires the schema first. So this is blocked until the schema is done.